apps/api/src/schema/catalogue.ts

```diff
@@ -0,0 +1,91 @@
+import { z } from "@hono/zod-openapi";
+
+export const sampleProgramsQuerySchema = z.object({
+  id: z.string().optional().openapi({
+    description: "Filter sample programs by program ID and returns all programs if omitted.",
+    example: "computerscience_bs",
+  }),
+});
+
+const STANDING_YEARS = ["Freshman", "Sophomore", "Junior", "Senior"] as const;
+
+export const standingyearEnum = z.enum(STANDING_YEARS);
```

Comment:
This isn't correct `camelCase` if you meant "standing" and "year" to be different years.

I also think the word "enum" isn't quite normative in this codebase; maybe "schema" is more appropriate.